src/main/webapp/WEB-INF/tags/student/registerMessage.tag

```diff
@@ -0,0 +1,10 @@
+<%@ tag description="Register message for unregistered students" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="googleId" required="true" %>
+<%@ attribute name="registerMessage" required="true" %>
+
+<c:if test="${empty googleId}">
+    <div id="registerMessage" class="alert alert-info">
+        ${registerMessage}
+    </div>
+</c:if>
```

Comment:
Right now only studentFeedbackResults.jsp is using it so I am transferring the contents to the JSP file and deleting this tag.